### DISCUSSION TITLE
How to delete and rename Space applications?

### DISCUSSION THREAD

**Discussion by begoon** [deta/cloud-docs#757] [General] [answered]
I cannot find anywhere in the UI or the `space` CLI how to rename or delete existing apps.

Is it possible?

**Comment by abdelhai** [comment, marked as answer]
you can change your app's name via the `name` attribute: https://deta.space/docs/en/reference/spacefile#name
if you mean the project name, then it can't be changed right now but soon.

**Comment by begoon** [comment]
Thanks for the answer.

The “name” attribute changes the name in the Space Dashboard but it doesn’t allow to change the URL prefix. 

I was thinking to drop the app all together and recreate it under a new name, but then I found that I cannot just delete the app.

**Reply by abdelhai** [reply]
ok, that's true. we took note and will be working on making this possible soon. for now, the best option is to start with a new builder project.

**Comment by begoon** [comment]
I guess even the ability to delete the app would help a lot if the renaming is tricky to implement.

**Reply by abdelhai** [reply]
yes. this should be coming soon! we were just focused on feature parity with deta.sh for the migration. but a lot f ux improvements are coming soon
